CMakeLists.txt

```diff
@@ -394,7 +394,16 @@ endif()
 # Build modcc flags
 #------------------------------------------------
 
-set(modcc $<TARGET_FILE:modcc>)
+if(ARB_MODCC)
+    find_program(modcc NAMES ${ARB_MODCC} NO_CMAKE_PATH NO_CMAKE_ENVIRONMENT_PATH NO_CMAKE_SYSTEM_PATH)
```

Comment:
Quick question: does this honour $PATH? The documentation is ... well ... par for CMake.

```
REQUIRED

    New in version 3.18.

    Stop processing with an error message if nothing is found, otherwise the search will be attempted again the next time find_program is invoked with the same variable.
```

Reply:
This PR is a revert, so I did not change these flags from what they were before. I must admit I don't understand the docs on some of them.

Are you saying we should add REQUIRED? That sounds extremely logical, although I can't see how you'd ever get the Arbor repo in that state.